Cargo.toml

```diff
@@ -18,6 +18,7 @@ inherits = "release"
 
 [dependencies]
 candid = { workspace = true }
+getrandom = { workspace = true }
```

Comment:
(Workaround for a `wasm-bindgen` issue that resurfaced after removing the unused Rust library)